src/components/badge.component.js

```diff
@@ -14,7 +14,7 @@ const BadgeContainer = styled.View`
   flex: 1;
   align-items: center;
   justify-content: center;
-  border-radius: 18;
+  border-radius: 25;
```

Comment:
normally, in order to have a circle, the border-radius should be equal to width & height.
could you provide a screenshot of the result?
don't hesitate if you need any assistance with this PR!